scripts/build-deb.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -x
+set -e
+
+PARALLEL=${PARALLEL:-1}
+TMP=$(mktemp -d /tmp/debuild.XXXXXX)
+
+function cleanup() {
+  [[ -d $TMP ]] && rm -rf $TMP
+}
+trap cleanup EXIT
+
+mkdir $TMP/bcc
+cp -a * $TMP/bcc
+pushd $TMP
+tar zcf bcc_0.1.6.orig.tar.gz bcc/
+cd bcc
+DEB_BUILD_OPTIONS="nocheck parallel=${PARALLEL}" debuild -us -uc
+popd
+
+cp $TMP/*.deb .
```

Comment:
Is it possible to run the generation of this file (scripts/build-deb.sh from scripts/build-deb.sh.in) by using a runtime command by buildbot instead of putting this file into the repository?